libs/payments/ui/src/lib/nestapp/validators/CheckoutCartWithStripeActionArgs.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { IsNumber, IsString } from 'class-validator';
+
+export class CheckoutCartWithStripeActionArgs {
+  @IsString()
```

Comment:
oh-so-random-question: When validating strings, should we also do some [`@MaxLength()` decorator](https://www.npmjs.com/package/class-validator#validation-messages) to ensure that strings aren't longer than their respective database column size? (although, I guess in this case it might be moot if `carts.id` is some `binary(16)` primary key.

Reply:
@julianpoy @StaberindeZA this is a good question. Either way, I'm curious of the answer. Maybe Peter is right and it's moot but would be great to know as I'm curious as well.